src/functionalTest/resources/application-functional.yaml

```diff
@@ -6,7 +6,3 @@ elasticsearch:
   hosts: ${ELASTICSEARCH_DATA_NODES_HOSTS:"http://localhost:9200"}
   request.timeout: ${ELASTICSEARCH_REQUEST_TIMEOUT:6000}
   cases.index.name.pattern: ${ELASTICSEARCH_CASES_INDEX_NAME_PATTERN:%s_cases}
-
-deletable.case.types: ${DELETE_CASE_TYPES:FT_MasterCaseType}
```

Comment:
Do you no longer need deletable case types?

Reply:
We still do need it but it's injected using a different strategy - please see [here](https://github.com/hmcts/ccd-case-disposer/blob/0c6a6a57ef2c26fac629e8cba94238685b8156b0/src/functionalTest/java/uk/gov/hmcts/reform/ccd/parameter/TestParameterResolver.java#L50) and [here](https://github.com/hmcts/ccd-case-disposer/blob/RDM-12856/src/functionalTest/java/uk/gov/hmcts/reform/ccd/parameter/TestParameterResolverTest.java)
This approach enables us change the `deletable.case.types (DELETABLE_CASE_TYPES_PROPERTY in functional test context)` at runtime.

Reply:
👍